### PR TITLE
[IOTDB-6280] Parallel RegionGroup activation

### PR DESCRIPTION
**Pull request by CRZbulabula**
https://issues.apache.org/jira/secure/RapidBoard.jspa?rapidView=492&view=detail&selectedIssue=IOTDB-6280&quickFilter=2189

The RegionGroup activation is now started simultaneously

image

### CODE REVIEW

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM~
The current code is consistent with the current style, but may need to be changed to an asynchronous style in the future to avoid synchronous sleep occupancy of threads

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/statemachine/CreateRegionGroupsProcedure.java [hunk 1]
```diff
@@ -173,40 +173,47 @@ protected Flow executeFromState(ConfigNodeProcedureEnv env, CreateRegionGroupsSt
         // Build RegionGroupCache immediately to make these successfully built RegionGroup available
         createRegionGroupsPlan
             .getRegionGroupMap()
+            .entrySet()
+            .parallelStream()
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/statemachine/CreateRegionGroupsProcedure.java:177]: I suggest we remove the parallelism here, as it's very unlikely that multiple sg regions will be created at the same time, but parallel here and below might introduce deadlock issues in some scenarios

**Review by OneSizeFitsQuorum — COMMENTED** [review]
PTAL

File: iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/statemachine/CreateRegionGroupsProcedure.java (hunk 1 above)
Comment (CRZbulabula) [reply iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/impl/statemachine/CreateRegionGroupsProcedure.java]: Fixed

**Pull request merged**